### PR TITLE
Add a router for app's routes

### PR DESCRIPTION
**Pull request by oleh-momot**
It needed to change the destination into `feature/routes` after `feature/template-routes` will be merged

### CODE REVIEW

File: registry/server/appRoutes/routes/updateAppRoute.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    const {
+        slots: appRouteSlots,
+        ...appRoute
+    } = req.body;
+    const appRouteId = req.params.id;
+
+    await db('routes').where('id', appRouteId).update(appRoute);
+
+    if (!_.isEmpty(appRouteSlots)) {
+        await db('route_slots').where('routeId', appRouteId).delete();
```
Comment (StyleT) [inline registry/server/appRoutes/routes/updateAppRoute.ts]: batchInsert & delete should be a part of the same transaction/query

Comment (StyleT) [reply registry/server/appRoutes/routes/updateAppRoute.ts]: Same goes to `db('routes').where('id', appRouteId).update(appRoute);`

Comment (oleh-momot) [reply registry/server/appRoutes/routes/updateAppRoute.ts]: Done

File: registry/server/appRoutes/routes/updateAppRoute.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    await validateRequestBeforeUpdateAppRoute(req, res);
+
+    const {
+        slots: appRouteSlots,
+        ...appRoute
+    } = req.body;
+    const appRouteId = req.params.id;
+
+    const appRoutes = await db.transaction(async (transaction) => {
+        try {
```
Comment (StyleT) [inline registry/server/appRoutes/routes/updateAppRoute.ts]: You don't need it.
Throwing an error directly from the transaction handler function automatically rolls back the transaction, same as returning a rejected promise.

Comment (oleh-momot) [reply registry/server/appRoutes/routes/updateAppRoute.ts]: Done

File: registry/server/appRoutes/routes/createAppRoute.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        await db.batchInsert('route_slots', _.compose(
+            _.map((appRouteSlotName) => _.compose(
+                stringifyJSON(['props']),
+                _.assign({ name: appRouteSlotName, routeId: appRouteId }),
+                _.get(appRouteSlotName)
+            )(appRouteSlots)),
+            _.keys,
+        )(appRouteSlots)).transacting(transaction);
+
+        const appRoutes = await db
```
Comment (StyleT) [inline registry/server/appRoutes/routes/createAppRoute.ts]: Move out of transaction

Comment (oleh-momot) [reply registry/server/appRoutes/routes/createAppRoute.ts]: Done

**Pull request merged**